2023-04-18 16:32:05,857 - distributed.nanny - INFO -         Start Nanny at: 'tcp://172.16.3.164:33208'
2023-04-18 16:32:06,250 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-worker-space/worker-u4lektge', purging
2023-04-18 16:32:06,599 - distributed.worker - INFO -       Start worker at:   tcp://172.16.3.164:40863
2023-04-18 16:32:06,599 - distributed.worker - INFO -          Listening to:   tcp://172.16.3.164:40863
2023-04-18 16:32:06,599 - distributed.worker - INFO -           Worker name:             SLURMCluster-0
2023-04-18 16:32:06,599 - distributed.worker - INFO -          dashboard at:         172.16.3.164:40912
2023-04-18 16:32:06,599 - distributed.worker - INFO - Waiting to connect to:    tcp://172.16.3.91:39170
2023-04-18 16:32:06,599 - distributed.worker - INFO - -------------------------------------------------
2023-04-18 16:32:06,599 - distributed.worker - INFO -               Threads:                         30
2023-04-18 16:32:06,599 - distributed.worker - INFO -                Memory:                  93.13 GiB
2023-04-18 16:32:06,599 - distributed.worker - INFO -       Local Directory: /tmp/dask-worker-space/worker-g2bptwj1
2023-04-18 16:32:06,599 - distributed.worker - INFO - -------------------------------------------------
2023-04-18 16:32:06,605 - distributed.worker - INFO -         Registered to:    tcp://172.16.3.91:39170
2023-04-18 16:32:06,605 - distributed.worker - INFO - -------------------------------------------------
2023-04-18 16:32:06,605 - distributed.core - INFO - Starting established connection to tcp://172.16.3.91:39170
/vortexfs1/home/anthony.meza/.conda/envs/notebook_env/lib/python3.10/site-packages/xarray/core/common.py:965: FutureWarning: 'base' in .resample() and in Grouper() is deprecated.
The new arguments that you should use are 'offset' or 'origin'.

>>> df.resample(freq="3s", base=2)

becomes:

>>> df.resample(freq="3s", offset="2s")

  grouper = pd.Grouper(
2023-04-18 16:33:22,789 - distributed.core - INFO - Event loop was unresponsive in Worker for 4.08s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2023-04-18 16:35:26,122 - distributed.core - INFO - Event loop was unresponsive in Worker for 12.82s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2023-04-18 16:35:36,384 - distributed.utils_perf - INFO - full garbage collection released 2.11 GiB from 0 reference cycles (threshold: 9.54 MiB)
2023-04-18 16:35:44,190 - distributed.core - INFO - Event loop was unresponsive in Worker for 18.07s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2023-04-18 16:35:50,165 - distributed.core - INFO - Event loop was unresponsive in Worker for 5.98s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
2023-04-18 16:36:35,437 - distributed.core - INFO - Event loop was unresponsive in Worker for 4.71s.  This is often caused by long-running GIL-holding functions or moving large chunks of data. This can cause timeouts and instability.
slurmstepd: error: *** JOB 3066586 ON pn114 CANCELLED AT 2023-04-18T16:36:56 ***
